src/Tables.jl

```diff
@@ -565,6 +565,29 @@ struct Partitioner{T}
     x::T
 end
 
+"""
+    getrows(x, inds; view=nothing)
+
+Return one or more rows from table `x` according to the position(s) specified by `inds`:
+
+- If `inds` is a single integer return a row object.
+- If `inds` is a collection of integers, return a table object. 
```

Comment:
```suggestion
- If `inds` is a collection of integers, return an indexable object of rows
```
Just a bit more specific; one of the main motivations for this was to ensure users can get an in-memory/indexable collection of rows in a consistent way.

Reply:
Are you sure we want to require `Tables.getrows` to return a indexable collection of rows? That would mean that e.g. for a `NamedTuple` of vectors `getrows` couldn't return another `NamedTuple` of vectors, even if that's the most efficient representation.